lib/monitor/monitor.go

```diff
@@ -18,31 +18,51 @@ import (
 var Sink = "0.0.0.0"
```

Comment:
As far as I understand a `Measure` is a single data point whereas a `Measurement` is a collections of `Measure`s? Maybe we can express this relation in the naming of the structs? The current naming scheme might lead to confusions.

(`Record` might be a suitable synonym for a single `Measure`.)